docs/zh/deploy/install_deploy.md

```diff
@@ -212,13 +160,11 @@ tar -zxvf openmldb-0.7.1-linux.tar.gz
 cd openmldb-0.7.1-linux
 ```
 
-### 配置
-#### 环境配置
+### 环境配置
 环境变量定义在`conf/openmldb-env.sh`，如下表所示：
 
 | 环境变量                              | 默认值                                | 定义                                                                      |
 |-----------------------------------|------------------------------------|-------------------------------------------------------------------------|
-| OPENMLDB_VERSION                  | 0.7.1                              | OpenMLDB版本                                                              |
```

Comment:
why delete the version

Reply:
this should be a typo... will recover this